packages/utils/src/schema/sanitizeDataForNewSchema.ts

```diff
@@ -137,7 +137,7 @@ export default function sanitizeDataForNewSchema<
     });
 
     newFormData = {
-      ...data,
+      ...(typeof data == 'string' || Array.isArray(data) ? {} : data),
```

Comment:
This would also work I believe...
```suggestion
      ...(typeof data == 'string' || Array.isArray(data) ? undefined : data),
```

Reply:
Also, can you update the `CHANGELOG.md`

Reply:
Hi @heath-freenome , Thanks for the feedback! Yes your suggestion will work as well, I'll add it to the PR. Also I haven't check the tests for this file, I'll add specific cases for this scenario if needed, that's why I kept the PR as a draft.